### ISSUE TITLE
Default value not working in v.4.3.0

### ISSUE THREAD

**Issue by gepd** [anubra266/choc-autocomplete#35]
I'm having problems to get the default value in `v4.3.0`, the problem is I'm not able to replicate it in the codeSandbox but when I switch to `v4.2.0` it's working, I have looked the changes in the lastest version and I saw you added a `defaultValue` prop

https://github.com/anubra266/choc-autocomplete/blob/932823f7935abe6ef0909594fe4fc58ee058bbf5/src/use-autocomplete.ts#L160

I put `console.log(autoCompleteProps.defaultValue)` and it's undefined so the return of `useParseDefaultValue` is undefined as well, as I'm using the input in controlled mode I think it should get `value` as default value, I think that is the way how this was working before

**Comment by anubra266** [comment]
Hi, can I see your code. @gepd

**Comment by anubra266** [comment]
Hi, can I see your code. @gepd

**Comment by gepd** [comment]
Im using RHF so the default value is given in 

```js
const methods = useForm({ defaultValues: document });
```

The component is declared like this:

```js
<FormControl isInvalid={methods.formState.errors[name]} mb="3">
  {isEdit && (
    <FormLabel htmlFor={name} mb="0" fontWeight="normal">
      {label}
    </FormLabel>
  )}
  <Controller
    name={name}
    control={methods.control}
    render={({ field: { onChange, onBlur, value } }) => (
      <AutoComplete {...props} freeSolo rollNavigation suggestWhenEmpty openOnFocus isReadOnly={!isEdit}>
        <AutoCompleteInput
          onChange={onChange}
          value={value}
          onBlur={onBlur}
          variant={isEdit ? "filled" : "ghost"}
          autocomplete="off"
        />
        <AutoCompleteList>
          <AutoCompleteGroup title="Title" showDivider>
            {options.map((option, oid) => (
              <AutoCompleteItem key={`option-${oid}`} value={option} textTransform="capitalize">
                {option}
              </AutoCompleteItem>
            ))}
          </AutoCompleteGroup>
        </AutoCompleteList>
      </AutoComplete>
    )}
  />
  <FormErrorMessage>{methods.formState.errors[name]?.message}</FormErrorMessage>
</FormControl>
```
I know the value is been passing because when I console log the `value` prop I can see it, but also the same code work without problem in `v4.2.0`

**Comment by anubra266** [comment]
Ah I get now. The `defaultValue` overrides the value you're providing to the input.

**Comment by gepd** [comment]
yes I think so!

**Comment by anubra266** [comment]
I added the `defaultValue` so you could define tags, beforeHand, I'll change it to defaultTags.

**Comment by anubra266** [comment]
Fixed in 4.3.1

**Comment by gepd** [comment]
Thanks It's working again!
